packages/gatsby-plugin-image/src/components/lazy-hydrate.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { MutableRefObject } from "react"
-import { hydrate, render } from "react-dom"
+// @ts-ignore - react 18 has hydrateRoot
+import { hydrate, render, createRoot } from "react-dom"
```

Comment:
Did you mean to use `hydrateRoot` or is the ts-ignore comment off?

Reply:
createRoot